api/base/views.py

```diff
@@ -129,7 +129,7 @@ def get_serializer_context(self):
         context.update({
             'enable_esi': (
                 is_truthy(self.request.query_params.get('esi', django_settings.ENABLE_ESI)) and
-                self.request.accepted_media_type in django_settings.ESI_MEDIA_TYPES
+                self.request.accepted_renderer.media_type in django_settings.ESI_MEDIA_TYPES
```

Comment:
I can't think of any way to write a test to accomplish this. If DRF changes their API ... I don't know how we can test against every potential change they could make. 

The point I was hoping to get across was that "tests passing" isn't a stamp of approval that the actual underlying functionality works. As we use more packages and frameworks that we didn't write we're going to have to pay more attention to functional, integration, and regression testing.

Unit testing is great for units of code but it won't find a integration problem that occurs in the final output (html/json) that is built using several different independent frameworks.